zimui/src/components/TopicHome.vue

```diff
@@ -272,4 +284,10 @@ const goToPreviousPage = () => {
 footer {
   text-align: center;
 }
+
+.error-message {
```

Comment:
Is this sufficient styling? I do not mind to have something minimalist, but here it seems very minimal. Please share a screenshot of the UI so that we can all get a better understanding of it.

Reply:
This is standard error message styling.

Reply:
This is rude, I have doubts about the styling, I asked to **Please share a screenshot** and you just resolve the conversation saying "everything is fine"?

Reply:
@benoit74 thank you, you made me push my limits.
Here is the screenshot of the error message

![Screenshot from 2024-04-09 23-04-26](https://github.com/openzim/kolibri/assets/137636943/6fd2e11c-cce5-418a-b055-7117d3218553)

Reply:
Thank you! This is pretty minimalist, but at least not ugly.
Does the back button of the navigator still works? e.g. if the user did opened a working topic before and then for any reason open a topic which is KO, is the user capable to navigate back to the page which did worked?

Reply:
Yes, the back button is working as it was before.
What do you think about displaying errors like the following screenshot?
![Screenshot from 2024-04-11 17-05-50](https://github.com/openzim/kolibri/assets/137636943/22133399-ae6e-44fd-beec-dbeaf207546a)